packages/blockchain-ui/src/theme/theme.ts

```diff
@@ -43,13 +43,28 @@ export const createTheme = ({
 const DefaultTheme = createTheme();
 
 const DarkTheme = createTheme({
+  components: {
+    Input: {
+      box: {
+        common: {
+          outlined: colors.grey500Opacity32,
```

Comment:
use alpha function and pass opacity as value here . it is present in utils/alpha.ts